code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -1,9 +1,9 @@
 
 //Called when the mob is hit with an item in combat.
-/mob/living/carbon/resolve_item_attack(obj/item/I, mob/living/user, var/effective_force, var/hit_zone)
```

Comment:
This effective force var seems to have been erroneously copied from the proc below it, and doesn't belong here.